src/platform_impl/linux/x11/window.rs

```diff
@@ -585,6 +585,16 @@ impl UnownedWindow {
 
     fn set_fullscreen_inner(&self, fullscreen: Option<Fullscreen>) -> Option<util::Flusher<'_>> {
         let mut shared_state_lock = self.shared_state.lock();
+        // Map Borderless(`None`) to current monitor.
+        let fullscreen = match fullscreen {
+            Some(Fullscreen::Borderless(None)) => {
+                let current_monitor = Some(RootMonitorHandle {
+                    inner: PlatformMonitorHandle::X(self.current_monitor()),
+                });
+                Some(Fullscreen::Borderless(current_monitor))
+            }
+            _ => fullscreen,
+        };
```

Comment:
Could also make it a `mut fullscreen` in parameters and use `if let Some` here, right? I think that might come out a couple lines shorter.

Reply:
I'll just match it later on instead of expanding early on